src/libs/actions/App.js

```diff
@@ -341,12 +341,18 @@ function openProfile(personalDetails) {
 }
 
 function beginDeepLinkRedirect() {
+    // Match any magic link (/v/<account id>/<6 digit code>)
+    const isMagicLink = CONST.REGEX.ROUTES.VALIDATE_LOGIN.test(window.location.pathname)
```

Comment:
We might as well move this below the anonymous if, since it's unnecessary if we hit that.

Reply:
Done!